system/security/LdapSecurity/permissions.cpp

```diff
@@ -60,14 +60,14 @@ PermissionProcessor::PermissionProcessor(IPropertyTree* config)
     m_cfg->getProp(".//@ldapAddress", m_server);
 }
 
-unsigned PermissionProcessor::ldap2sec(unsigned ldapperm)
+SecAccessFlags PermissionProcessor::ldap2sec(unsigned ldapperm)
 {
     unsigned permission = 0;
 
     if((ldapperm & 0xFF) == 0xFF)
     {
         permission |= SecAccess_Full;
```

Comment:
I must be missing something here, it seems permission is set to 0, and then it is always assigned to the ( itselsef | SecAccess_Full).... why not just declare it as a secaccessflags and assign it directly to SecAccess_Full here? Also, wouldn't cast it to SecAccessFlags at the return in this scope